src/GovUk.Education.ExploreEducationStatistics.Public.Data.Processor.Tests/Functions/ProcessCompletionOfNextDataSetVersionImportFunctionTests.cs

```diff
@@ -117,6 +124,3168 @@ private static Mock<TaskOrchestrationContext> DefaultMockOrchestrationContext(Gu
+        {
+            var oldIndicatorMeta = DataFixture.DefaultIndicatorMeta()
+                .ForIndex(0, s => s.SetPublicId(SqidEncoder.Encode(1)))
+                .ForIndex(1, s => s.SetPublicId(SqidEncoder.Encode(2))) // Indicator deleted
+                .ForIndex(2, s => s.SetPublicId(SqidEncoder.Encode(3))) // Indicator deleted
+                .ForIndex(3, s => s.SetPublicId(SqidEncoder.Encode(4)))
+                .ForIndex(4, s => s.SetPublicId(SqidEncoder.Encode(5)))
+                .GenerateList(5);
+
+            var (originalVersion, _) = await CreateDataSetInitialVersion(oldIndicatorMeta);
+
+            var newIndicatorMeta = DataFixture.DefaultIndicatorMeta()
+                .ForIndex(0, UnchangedIndicatorMetaSetter(oldIndicatorMeta[0])) // Indicator unchanged
+                .ForIndex(1, s => s.SetPublicId(SqidEncoder.Encode(4))) // Indicator changed
+                .ForIndex(2, s => s.SetPublicId(SqidEncoder.Encode(5))) // Indicator changed
+                .ForIndex(3, s => s.SetPublicId(SqidEncoder.Encode(6))) // Indicator added
+                .ForIndex(4, s => s.SetPublicId(SqidEncoder.Encode(7))) // Indicator added
+                .GenerateList(5);
+
+            var (newVersion, instanceId) = await CreateDataSetNextVersion(
+                originalVersion: originalVersion,
+                indicatorMeta: newIndicatorMeta);
+
+            await CreateChanges(instanceId);
+
+            var changes = await GetIndicatorMetaChanges(newVersion);
+
+            // 2 Indicator changes
+            Assert.Equal(6, changes.Count);
+            Assert.All(changes, c => Assert.Equal(newVersion.Id, c.DataSetVersionId));
+
+            var oldIndicatorMetas = originalVersion.IndicatorMetas
+                .ToDictionary(m => m.PublicId);
+
+            var newIndicatorMetas = newVersion.IndicatorMetas
+                .ToDictionary(m => m.PublicId);
+
+            // Indicator deleted
+            AssertSingleIndicatorDeleted(changes, oldIndicatorMetas[SqidEncoder.Encode(2)]);
+
+            // Indicator deleted
+            AssertSingleIndicatorDeleted(changes, oldIndicatorMetas[SqidEncoder.Encode(3)]);
+
+            // Indicator changed
+            AssertSingleIndicatorChanged(
+                changes: changes,
+                expectedOldIndicatorMeta: oldIndicatorMetas[SqidEncoder.Encode(4)],
+                expectedNewIndicatorMeta: newIndicatorMetas[SqidEncoder.Encode(4)]);
+
+            // Indicator changed
+            AssertSingleIndicatorChanged(
+                changes: changes,
+                expectedOldIndicatorMeta: oldIndicatorMetas[SqidEncoder.Encode(5)],
+                expectedNewIndicatorMeta: newIndicatorMetas[SqidEncoder.Encode(5)]);
+
+            // Indicator added
+            AssertSingleIndicatorAdded(changes, newIndicatorMetas[SqidEncoder.Encode(6)]);
+
+            // Indicator added
+            AssertSingleIndicatorAdded(changes, newIndicatorMetas[SqidEncoder.Encode(7)]);
+        }
```

Comment:
Do we need this test case? We've got the test case lower down that checks the ordering and is testing the same things so it seems redundant to have this.

Reply:
yeah this one is redundant